readme.md

```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Artem-Bondarchuk.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Artem-Bondarchuk.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add all these links to the PR description

Reply:
Consider making a deploy 
<img width="836" alt="Screenshot 2024-08-16 at 15 06 06" src="https://github.com/user-attachments/assets/2785a131-8f17-47c7-94e1-2703099f770c">